database/seeders/PermissionsSeeder.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Database\Seeders;
+
+use App\Enums\Permission;
+use Illuminate\Contracts\Container\BindingResolutionException;
+use Illuminate\Database\Seeder;
+use Spatie\Permission\Models\Permission;
```

Comment:
on this model, you can use an alias for fix it

Reply:
good catch mate